### ISSUE TITLE
Reference sequence can't contain degenerate/heterozygous bases

### ISSUE THREAD

**Issue by zygoballus** [gear-genomics/pearl#26]
If the reference sequence contains a degenerate/heterozygous base (W, S, M, K, R, Y, etc.), it gives the error "FASTA file contains nucleotides != [ACGTN]."

As a quick and dirty fix, it could map these bases to N. The best fix would be to use the bases according to their meaning, e.g. C or T for Y.

**Comment by untergasser** [comment]
Yes, your quick fix is the intended way to use Pearl. The tool should be used to confirm nucleotide sequences, so degenerate/heterozygous bases will not solve the sequence.

Best,
Andreas
